syntax = "proto3";

package algotrader.model;



message DataSeries{

    message Item{
        int64 timestamp = 1;
        map<string, double> data = 3;
    }
    string series_id = 1;
    string name = 2;
    string desc = 3;
    repeated string inputs = 4;
    repeated string keys = 5;
    string default_output_key = 6;
    double missing_value_replace = 7;
    int64 start_time = 8;
    int64 end_time = 9;
    repeated Item items = 10;
}

